### DISCUSSION TITLE
Accessing store within Vue file

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11076] [General - Components / Directives / etc] [answered]
When accessing the store within a view file which of the following is the correct way of doing it? They both work but I've seen both being used. Not sure if their is a difference.

Option 1
```
import { defineComponent, computed, onMounted, ref } from "vue";
import { useStore } from "vuex";

export default defineComponent({
  name: "MainLayout",

  setup() {
    const store = useStore();
    const myData = computed(() => store.state.app.mydata);
  }
});
```

Option 2
```
import { defineComponent, computed, onMounted, ref } from "vue";
import { useStore } from "vuex";

export default defineComponent({
  name: "MainLayout",

  setup() {
    const $store = useStore();
    const myData = computed(() => $store.state.app.mydata);
  }
});
```

**Comment by hawkeye64** [comment, marked as answer]
The only difference in the above two examples is the name of the variable receiving the store. In other words, they are both exactly the same thing. Some people like to use the dollar sign to indicate it's an external library. That's it.

**Reply by JokerMartini** [reply]
thanks for clarifying. i tried searching online and felt really dumb doing so lol
